charts/sonarqube/templates/deployment.yaml

```diff
@@ -102,6 +102,9 @@ spec:
       {{- end }}
       {{- end }}
       containers:
+        {{- if .Values.extraContainers }}
+        {{- toYaml .Values.extraContainers | nindent 8 }}
+        {{- end }}
```

Comment:
Additional containers should be configured after the primary container.

Reply:
Done, moved below primary container